treeherder/model/management/commands/import_reference_data.py

```diff
@@ -41,5 +42,86 @@ def handle(self, *args, **options):
                         'active_status': machine_platform['active_status']
                     })
 
-        # TODO: Implement other types: product, build platform, repository groups,
-        # repositories, machine, job group, job type, failure classification
+        # machine
+        for machine in c.get_machines():
+            Machine.objects.get_or_create(
+                    id=machine['id'],
+                    name=machine['name'],
+                    defaults={
+                        'first_timestamp': machine['first_timestamp'],
+                        'last_timestamp': machine['last_timestamp'],
+                        'active_status': machine['active_status']
+                    })
+
+        # job group
+        for job_group in c.get_job_groups():
+            JobGroup.objects.get_or_create(
+                    id=job_group['id'],
+                    symbol=job_group['symbol'],
+                    name=job_group['name'],
+                    defaults={
+                        'description': job_group['description'],
+                        'active_status': job_group['active_status']
+                    })
+
+        # job type
+        for job_type in c.get_job_types():
+            jgroup = JobGroup.objects.filter(id=job_type['job_group'])
```

Comment:
Sorry, I missed this earlier: this should be a get, not a filter. filter returns an array of results.

So you want `jgroup = JobGroup.objects.get(id=job_type['job_group'])`